### PR TITLE
add catalog

### PR DESCRIPTION
**Pull request by MartaChobaniuk**
[DEMO LINK](https://martachobaniuk.github.io/layout_catalog/)
- [TEST REPORT LINK](https://martachobaniuk.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Oh! Good job!

Some comments left

File: src/styles/blocks/cards.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+.product-card {
+  @include fonts (10px, 14px, 400);
+  @include flex (flex, false, false, column);
+  width: $card-size;
+  background: $color-white;
+  color: $color-secondary;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: $card-margin-bottom;
```
Comment (slavanosatskyi) [inline src/styles/blocks/cards.scss:10]: do not use external geometry (margins in your case) on a block level elements

File: src/styles/reset.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```
[the remark this answers is not in the document]
Comment (slavanosatskyi) [reply src/styles/reset.scss:5]: > it's allowed to use only `box-sizing: border-box for` `*` selector

this comment is not fixed yet

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
Almost there! fix some comments from prev review + one additional comment from me

File: src/styles/blocks/cards.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  &__name {
+    @include fonts (12px, 18px, 500);
+    margin-top: 40px;
+    color: $color-main-accent;
+  }
+
+  &__code {
+    @include fonts (10px, 14px, 400);
```
Comment (YegorKorenkov) [inline src/styles/blocks/cards.scss:26]: font-weight 400 by default, redundant

File: src/styles/blocks/cards.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  &__rating {
+    @include flex (flex, false, space-between, row);
+    margin-top: 16px;
+    line-height: 16px;
+  }
+
+  &__rating-rewiews {
+    @include fonts (10px, 14px, 400);
```
Comment (YegorKorenkov) [inline src/styles/blocks/cards.scss:38]: same

**Review by YegorKorenkov — APPROVED** [review]
Nice work! Please dont forget re-deploy your demo, because now it's looks
image

**Pull request closed without merging**